Makefile

```diff
@@ -286,19 +286,55 @@ release-unix: clean full
 	fi
 
 #
-# make release-windows - Produces a binary release tarball containing teleport,
-# tctl, and tsh.
+# make release-windows-unsigned - Produces a binary release archive containing only tsh.
 #
-.PHONY:
-release-windows: clean all
+.PHONY: release-windows-unsigned
+release-windows-unsigned: clean all
 	@echo "---> Creating OSS release archive."
 	mkdir teleport
 	cp -rf $(BUILDDIR)/* \
 		README.md \
 		CHANGELOG.md \
 		teleport/
-	mv teleport/tsh teleport/tsh.exe
+	mv teleport/tsh teleport/tsh-unsigned.exe
 	echo $(GITTAG) > teleport/VERSION
+	zip -9 -y -r -q $(RELEASE)-unsigned.zip teleport/
+	rm -rf teleport/
+	@echo "---> Created $(RELEASE)-unsigned.zip."
+
+#
+# make release-windows - Produces an archive containing a signed release of
+# tsh.exe
+#
+.PHONY: release-windows
+release-windows: release-windows-unsigned
```

Comment:
Depend on `$(RELEASE)-unsigned.zip` instead of the .PHONY `release-windows-unsigned` so that we don't force a clean and rebuild every time we sign. E.g:

```make
release-windows-unsigned: $(RELEASE)-unsigned.zip

$(RELEASE)-unsigned.zip: clean all
	@echo "---> Creating OSS release archive."
        ...

release-windows: $(RELEASE)-unsigned.zip
	@if [ -z "$${WINDOWS_SIGNING_CERT}" ]; then
        ...
```

Reply:
Never mind -- this won't work without also fixing `$(RELEASE)-unsigned.zip`'s dependencies, which is out of scope here.